test/Microsoft.ML.Tests/TrainerEstimators/SymSgdClassificationTests.cs

```diff
@@ -20,5 +26,42 @@ public void TestEstimatorSymSgdClassificationTrainer()
             TestEstimatorCore(pipe, dataView);
             Done();
         }
+
+        [Fact]
+        public void TestEstimatorSymSgdInitPredictor()
+        {
+            (var pipe, var dataView) = GetBinaryClassificationPipeline();
+            var transformedData = pipe.Fit(dataView).Transform(dataView);
+
+            // No actual training happens. 
+            var args = new LinearClassificationTrainer.Arguments();
+            var initPredictor = new LinearClassificationTrainer(Env, args, "Features", "Label").Fit(transformedData);
+            var data = initPredictor.Transform(transformedData);
+
+            var withInitPredictor = new SymSgdClassificationTrainer(Env, "Features", "Label").Train(transformedData, initialPredictor: initPredictor.Model);
+            var outInitData = withInitPredictor.Transform(transformedData);
+
+            var notInitPredictor = new SymSgdClassificationTrainer(Env, "Features", "Label").Train(transformedData);
+            var outNoInitData = notInitPredictor.Transform(transformedData);
+
+            int numExamples = 10;
+            var col1 = data.GetColumn<float>(Env, "Score").Take(numExamples).ToArray();
+            var col2 = outInitData.GetColumn<float>(Env, "Score").Take(numExamples).ToArray();
+            var col3 = outNoInitData.GetColumn<float>(Env, "Score").Take(numExamples).ToArray();
+
+            bool col12Diff = default;
```

Comment:
you can also omit it for primitives.

Reply:
For some reason I then get warned that I use a unassigned variable.